cprnc_py/netcdf/netcdf_variable_scipy.py

```diff
@@ -47,11 +47,33 @@ def _get_data_from_slices(self, dim_slices):
         dim_slices should match the dimensionality of this variable
         """
 
-        # FIXME(wjs, 2015-12-31) Will this work on scalar data?
         if len(dim_slices) > 0:
-            vardata = self._var[dim_slices].copy()
+            # TODO(mfd, 2016-09-19) Find a better way of doing this
```

Comment:
I believe that's this commit comment: Verify that the shapes of a variable allow unraveling indexes before attempting to use them
This occurs when the dimension ranges are different for the two variables, which while probably not a standard use case for cprnc, does occur when running on unrelated files.